package.json

```diff
@@ -57,7 +57,7 @@
     "rxjs": "6.x"
   },
   "devDependencies": {
-    "@openmrs/esm-framework": "next",
+    "@openmrs/esm-framework": "latest",
```

Comment:
changed to `latest` because `next` has the `4.0 (React 18, Router 6 and Carbon 11)` changes merged in.